code/modules/integrated_electronics/subtypes/converters.dm

```diff
@@ -168,6 +202,56 @@
 
 	activate_pin(2)
 
+/obj/item/integrated_circuit/converter/findstring
+	name = "find text"
+	desc = "This gives position of sample in the string. Or returns 0."
```

Comment:
`string, or returns 0.`
Returning 0 from a search function is a bad idea unless you index the searched body from 1. Most code defaults the index for searches to 0, so if you get a 0 returned, your stuff was found at the very beginning.
With this, it'd either be at the beginning or nowhere at all, which is a bad uncertainty. I'd return -1 as the not-found value, because nothing has an element at position -1.

Reply:
Or byond is a heretic and indexes from 1, disregard the return-value stuff

Reply:
Yes,byond is heretic.